Aptoide/aptoide-client-v8
# Feature/asv 1076 desing specs for comments in store

**What does this PR do?**

This PR aims to finish the comment system, both on ui and ux levels. It also builds upon the previous comments development with bugfixes and some flow changes.

Still missing behaviour for when someone tries to comment while logged out;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] CommentFragment.java
- [ ] StoreLatestCommentsWidget.java

**How should this be manually tested?**

Try and use the comments on 3 levels: store, first level comment fragment (press the store comment widget more button), and second level comment fragment (replies). There is supposed to be only one level of depth in replies (reply only to main comment, no replies to replies). Also check the design matches zeplin;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1076](<https://aptoide.atlassian.net/browse/ASV-1076>)

**Code Review Checklist**
